sound/soc/sdw_utils/soc_sdw_rt712_sdca.c

```diff
@@ -1,8 +1,10 @@
 // SPDX-License-Identifier: GPL-2.0-only
+// This file incorporates work covered by the following copyright notice:
 // Copyright (c) 2023 Intel Corporation
+// Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
please check your copyright, it sometimes says All rights reserved and sometimes it doesn't.

Reply:
will fix it.

Reply:
Fixed.